src/content/self-hosted/manage/air-gapped.mdx

```diff
@@ -0,0 +1,136 @@
+---
+title: Air-Gapped Networks
+description: Installing Okteto in Air-Gapped Networks Environments
+sidebar_label: Air-Gapped Networks
+id: air-gapped
+---
+
+import variables from '../../variables.json';
+import CodeBlock from '@theme/CodeBlock';
+import TiersList from "@theme/TiersList";
+
+# Air-Gapped Networks Environments<TiersList tiers="Enterprise Self-Hosted" />
+
+Okteto works in air-gapped network environments, but will require some custom configuration.
+This guide has been crafted to help you with your air-gapped installation, but we recommend that you [reach out to our team](https://support.okteto.com/hc/en-us) to discuss your requirements and scenarios.
+
+## Disable Okteto Telemetry
+
+By default, Okteto sends [telemetry](self-hosted/helm-configuration.mdx#telemetry) to help us help us better understand how our customers use Okteto, as well as to help us prioritize fixes and features.
+But you can opt-out from this behavior by adding the following section to your Okteto Helm configuration:
+
+```yaml
+telemetry:
+  enabled: false
+```
+
+When you disable our telemetry, Okteto CLI analytics for all your developers are automatically disabled.
+
+## Host Okteto Images in your Registry
+
+By default, all the images needed to deploy the Okteto Platform, and the images used by the Okteto CLI, are hosted in DockerHub.
+If you need to host our images in your Private Registry, we recommend to use the same repository name and tags in your Private Registry.
+
+:::info
+This guide assumes that your cluster nodes have pull access to your private container registry. If that's not the case, please follow your infrastructure provider's instructions on how to do it
+:::
+
+For the Okteto Chart {variables.chartVersion} installation, these are the images you need to push to your Private Registry:
+
+- **okteto/backend:{variables.chartVersion}**
+- **okteto/frontend:{variables.chartVersion}**
+- **okteto/buildkit:{variables.chartVersion}**
+- **okteto/buildkit:{variables.chartVersion}-rootless**
+- **okteto/registry:{variables.chartVersion}**
+- **okteto/pipeline-installer:{variables.chartVersion}**
+- **okteto/pipeline-runner:{variables.chartVersion}**
+- **okteto/daemon:{variables.chartVersion}**
```

Comment:
I would add okteto/bin for the initContainer for okteto up. I added a FR to be able to add this image centralized and not in every manifest, I don't know if there is an ETA for it anyway :)

Reply:
okteto/bin and okteto/busybox will be unified with okteto/okteto